src/main/java/com/google/jenkins/plugins/computeengine/CleanLostNodesWork.java

```diff
@@ -39,11 +40,15 @@
 @Symbol("cleanLostNodesWork")
 public class CleanLostNodesWork extends PeriodicWork {
     protected final Logger logger = Logger.getLogger(getClass().getName());
+    public static final String NODE_IN_USE_LABEL_KEY = "jenkins_node_in_use";
+    public static final long RECURRENCE_PERIOD = Long.parseLong(
+            System.getProperty("jenkins.cloud.gcp.cleanLostNodesWork.recurrencePeriod", String.valueOf(HOUR)));
```

Comment:
To avoid arbitrary property names,
```suggestion
            System.getProperty(CleanLostNodesWork.class.getName() + ".recurrencePeriod", String.valueOf(HOUR)));
```